src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/RecapitiPersonaFisicaTest.java

```diff
@@ -686,18 +686,45 @@ public void nelPopUpEliminaIndirizzoPecSiCliccaSulBottoneConferma() {
         }
     }
 
+
+    @And("Nel pop up elimina indirizzo pec si clicca sul bottone annulla")
+    public void nelPopUpEliminaIndirizzoPecSiCliccaSulBottoneAnnulla() {
+        logger.info("Si clicca sul bottone annnulla");
+        if (recapitiDestinatarioPage.waitLoadPopUpElimina().equalsIgnoreCase("Rimuovi PEC")) {
+            recapitiDestinatarioPage.clickSuAnnullaElimina();
+        } else {
+            recapitiDestinatarioPage.clickSuChiudiPopUp();
+            recapitiDestinatarioPage.eliminaNuovaPec();
+            recapitiDestinatarioPage.eliminaPecEsistente();
+        }
+    }
```

Comment:
This one already exist:
https://github.com/pagopa/pn-frontend-e2e/blob/1f3209a14ec44180d6b619fe22f81eb56ce131e9/src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/RecapitiPGPagoPaTest.java#L99-L103